JellyfinPlayer/Components/PortraitItemView.swift

```diff
@@ -74,7 +74,7 @@ struct PortraitItemView: View {
                         .font(.caption)
                         .fontWeight(.medium)
                 } else {
-                    Text("S\(String(item.parentIndexNumber ?? 0)):E\(String(item.indexNumber ?? 0))")
+                    Text("S\(String(item.parentIndexNumber ?? 0)):E\(String(item.indexNumber ?? 0)) - \(item.name ?? "")")
```

Comment:
too long, most show episode names are > 20 characters, this will just look weird